syncd/CommandLineOptions.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include "syncd.h"
+
+#include <string>
+
+#define DEF_STRING_SAI_START_TYPE_COLD_BOOT         "cold"
```

Comment:
>DEF_STRING_SAI_START_TYPE_COLD_BOOT [](start = 8, length = 35)

Just call it `STRING_SAI_START_TYPE_COLD_BOOT` ?

Reply:
also woks, i can update

Reply:
done